llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -774,9 +773,9 @@ define void @foo(ptr %arg0, ptr %arg1) {
   EXPECT_EQ(NewLd->getName(), "NewLd");
 
   sandboxir::LoadInst *NewVLd =
-      sandboxir::LoadInst::create(Vld->getType(), Arg1, Align(8),
-                                  /*InsertBefore=*/Ret, Ctx,
-                                  /*IsVolatile=*/true, "NewVLd");
+      sandboxir::LoadInst::create(VLd->getType(), Arg1, Align(8),
+                                  /*InsertBefore=*/Ret,
+                                  /*IsVolatile=*/true, Ctx, "NewVLd");
 
   // Checking if create() was volatile
   EXPECT_TRUE(NewVLd->isVolatile());
```

Comment:
So far we have tested `create(..., /*InsertBefore=*/)`, `create(..., /*InsertBefore=*/, /*IsVolatile=*/)` but we have not tested the ones with /*InsertAtEnd=*/. Could you add tests for these too?

Reply:
I added tests for it